src/main/java/org/terasology/simpleFarming/systems/BushAuthoritySystem.java

```diff
@@ -170,25 +189,22 @@ private EntityRef doBushGrowth(BushDefinitionComponent bushComponent, int stages
      * the bush's {@link BushDefinitionComponent#sustainable sustainable} value.
      *
      * @param event  the activation event
-     * @param entity the entity doing the harvesting
+     * @param entity the block entity
      */
     @ReceiveEvent
-    public void onHarvest(ActivateEvent event, EntityRef entity) {
-        EntityRef target = event.getTarget();
-        EntityRef harvester = entity.equals(target) ? event.getInstigator() : entity;
-        if (!event.isConsumed() && areValidHarvestEntities(target, harvester)) {
-
-            BushDefinitionComponent bushComponent = target.getComponent(BushDefinitionComponent.class);
+    public void onHarvest(ActivateEvent event, EntityRef entity, BushDefinitionComponent bushComponent, BlockComponent blockComponent) {
+        EntityRef harvester = event.getInstigator();
+        if (!event.isConsumed() && areValidHarvestEntities(entity, harvester)
+                // only let player characters harvest to avoid item use triggering a harvest
+                && harvester.hasComponent(PlayerCharacterComponent.class)) {
```

Comment:
This stops functionality like the Harvester Cart in Additional Rails. I think it should be possible for things other than the player to be able to harvest the bush

Reply:
You are right.  I had some other problem that caused this behavior.  Thanks for the sanity check.  <removed>